modules/users/client/views/admin/edit-user.client.view.html

```diff
@@ -20,7 +20,7 @@
           </div>
         </div>
         <div class="form-group" show-errors>
-          <label class="control-label" for="roles">Roles</label>
+          <label class="control-label" for="roles">Roles</label><!--Someone should make this one a dropdown box -->
```

Comment:
Create a user story with all of the details of what you were thinking

Reply:
It's fine to leave a comment in the codebase for now, in fact we should. Not sure if you're telling me I should take this out but it's staying.

Reply:
I'm just saying that pivotal tracker is the best place to record these because we can prioritize them, assign them, and go into greater detail.